### PR TITLE
Hello world example - minor update

### PR DESCRIPTION
**Pull request by duboisp**
Separating and improving how to support plugin configuration into an additional step. Also it apply the suggestion from @LaurentGoderre in PR#7801

### CODE REVIEW

File: src/other/helloworld/demo/bonjour.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ var nomComposant = "wb-bonjour",
 	selecteur = "." + nomComposant,
 	initEvent = "wb-init" + selecteur,
 	$document = wb.doc,
+	defaut = {},
```
Comment (LaurentGoderre) [inline src/other/helloworld/demo/bonjour.js]: Knit picking but this is misspelt. Can you also make it plurals for consistency? `defaults`

I notice that some are plural and some ar singular but all seem to be missing the L

**Review by LaurentGoderre — CHANGES_REQUESTED** [review]

File: src/other/helloworld/demo/bonjour.js (hunk 1 above)
Comment (RobJohnston) [inline src/other/helloworld/demo/bonjour.js]: Speaking of... it's nitpicking, not knit picking. :-) ;-)
https://en.wikipedia.org/wiki/Nitpicking

Comment (duboisp) [inline src/other/helloworld/demo/bonjour.js]: no prob, I will fix it

Comment (LaurentGoderre) [inline src/other/helloworld/demo/bonjour.js]: Well played @RobJohnston .... well played!

**Comment by LaurentGoderre** [comment]
@duboisp you're still missing the "L" in defaults

**Comment by duboisp** [comment]
@LaurentGoderre le mot "défaut" ne prend pas de "L" en français.

**Comment by LaurentGoderre** [comment]
But the rest of the plugins use english variables names

**Comment by duboisp** [comment]
Ça ne devrais pas pour la version française "bonjour.js". Je remarque que j'ai oublié de traduire le mot "Event", je vais mettre à jour mon PR ce soir.

File: src/other/helloworld/demo/helloworld.js [hunk 2]
```diff
@@ -37,7 +38,13 @@ var componentName = "wb-helloworld",
 			// ... Do the plugin initialisation
 
 			// Get the plugin JSON configuration set on attribute data-wb-helloworld
-			settings = wb.getData( $elm, componentName );
+			settings = $.extend(
+				true,
+				{},
+				defauts,
```
Comment (LaurentGoderre) [inline src/other/helloworld/demo/helloworld.js]: It's missing the "l" here in the english one

**Comment by LaurentGoderre** [comment]
I'm not sure that we should be translating vartiable names but ok...

**Comment by duboisp** [comment]
@LaurentGoderre I updated the hello world example by considering your previous comments/suggestion. Thanks for that. 

Did you have any other comments in regards of this PR that needs to be done before to get it merged?

**Comment by LaurentGoderre** [comment]
Looks good to me

**Pull request merged**